src/Features/CSharp/Portable/MakeLocalFunctionStatic/MakeLocalFunctionStaticCodeFixHelper.cs

```diff
@@ -155,7 +133,7 @@ private static bool CanMakeLocalFunctionStatic(ImmutableArray<ISymbol> captures)
                         {
                             syntaxEditor.ReplaceNode(
                                 identifierNode,
-                                (node, generator) => generator.IdentifierName(parameter.Name.ToIdentifierToken()).WithTriviaFrom(node));
```

Comment:
Reverted this change in last commit.

Reply:
unfortunate if they have different behavior.  i want your change here to be good :)

Reply:
Tests passed either way, just reverted as wasn't necessary.